Modules/Challenges/test/Challenges.API.Tests.Integration/Features/CreateChallenge.feature

```diff
@@ -0,0 +1,36 @@
+﻿@Integration
+Feature: Create a Challenge
+
+User creates a Challenge
+
+Scenario: Creating a Challenge as an anonymous user
+	Given the user is unauthenticated
+	When a POST request is sent to the Challenges endpoint
+	Then the response status code is 201
+	And the response contains a Challenge
+
+Scenario: Creating a Challenge as an authenticated user
+	Given the user is authenticated
+	When a POST request is sent to the Challenges endpoint
+	Then the response status code is 201
+	And the response contains a Challenge
+
+Scenario: Creating a Challenge with a JSON sent in the request content
+	When a POST request is sent to the Challenges endpoint with
+		| Key  | Value                              |
+		| Body | {"this": "is some arbitrary json"} |
+	Then the response status code is 201
+	And the response contains a Challenge
```

Comment:
We should add `And the Challenge does not contain information about the creator`, which asserts that createdBy and createadByDevice are null.